signature/policy_eval_sigstore.go

```diff
@@ -89,16 +101,19 @@ type sigstoreSignedTrustRoot struct {
 func (pr *prSigstoreSigned) prepareTrustRoot() (*sigstoreSignedTrustRoot, error) {
 	res := sigstoreSignedTrustRoot{}
 
-	publicKeyPEM, err := loadBytesFromConfigSources(configBytesSources{
+	publicKeyPEMs, err := loadBytesFromConfigSources(configBytesSources{
 		inconsistencyErrorMessage: `Internal inconsistency: both "keyPath" and "keyData" specified`,
 		path:                      pr.KeyPath,
 		data:                      pr.KeyData,
 	})
 	if err != nil {
 		return nil, err
 	}
-	if publicKeyPEM != nil {
-		pk, err := cryptoutils.UnmarshalPEMToPublicKey(publicKeyPEM)
+	if publicKeyPEMs != nil {
+		if len(publicKeyPEMs) != 1 { // Coverage: We only provide single-element sources to loadBytesFromConfigSources, and at most one is allowed.
```

Comment:
nit: I guess the comment is telling something like "Should never happen", but my tiny brains fail to read it this way.

Reply:
“Coverage:” is, in this subpackage, a shorthand for “this line is not covered by unit tests because …”.

Added an explicit “this should never happen” to the start of the explanation, hopefully that helps.